### PR TITLE
Surface page

### PR DESCRIPTION
**Pull request by AnthonyStembreit**
I have detailed what I have done in the Cerbanimo general chat in discord. Let me know if you have any questions!

### CODE REVIEW

File: client/src/components/outerNav/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+
+export default function OuterNav() {
+  return (
+    <Navbar id="outsideNav" >
+      <Navbar.Brand>Cerbanimo</Navbar.Brand>
+      <Nav className="mr-auto">
+        <Link to="/home"
+          className={
+            window.location.pathname === "/" || window.location.pathname === "/home"
```
[the remark this answers is not in the document]
Comment (teXupport) [reply client/src/components/outerNav/index.js:14]: I may just need to see it in action. :)

File: client/src/pages/register/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export default function Register(){
+    return(
+        <div>
+            Home
```
Comment (teXupport) [inline client/src/pages/register/index.js]: I assume you'll be adding more anyway, but noticed you're sending back "Home" for the Register. Won't hold up the review, just wanted to point it out.

File: client/src/components/outerNav/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (AnthonyStembreit) [reply client/src/components/outerNav/index.js:14]: issues resolved on a different branch will be in next pull request :)

[the remark this answers is not in the document]
Comment (teXupport) [reply client/src/components/outerNav/index.js:14]: Sounds good. You're good to merge this PR then. Thanks!

**Review by teXupport — APPROVED** [review]

**Pull request merged**